src/main/resources/static/js/subject.js

```diff
@@ -0,0 +1,16 @@
+function onButtonClick(){
```

Comment:
why is this called subject.js but then you call the function addButtonClick and the id #addBtn? The naming does not make any sense. What exactly is this? Name is accordingly.